demos/supabase-todolist/lib/attachments/queue.dart

```diff
@@ -39,10 +39,11 @@ class PhotoAttachmentQueue extends AbstractAttachmentQueue {
   }
 
   @override
-  Future<Attachment> savePhoto(String photoId, int size) async {
-    String filename = '$photoId.jpg';
+  Future<Attachment> saveFile(String fileId, int size) async {
+    String filename = '$fileId.jpg';
```

Comment:
This still seems like it would be for photo uploads. Should the `extension` and `mediaType` not be configurable or passed in? It could be defaulted to an image.

Reply:
Yeah nice catch!